common/test/acceptance/tests/lms/test_problem_types.py

```diff
@@ -801,6 +807,29 @@ def answer_problem(self, correctness):
         self.problem_page.fill_answer(second_addend, input_num=1)
 
 
+class JSInputTypeTest(ProblemTypeTestBase, ProblemTypeA11yTestMixin):
+    """
+    TestCase Class for jsinput (custom JavaScript) problem type.
+    Right now the only test point that is executed is the a11y test.
+    This is because the factory simply creates an empty iframe.
+    """
+    problem_name = 'JSINPUT PROBLEM'
+    problem_type = 'customresponse'
+
+    factory = JSInputXMLFactory()
+
+    factory_kwargs = {
+        'question_text': 'IFrame shows below (but has no content)'
+    }
+
+    def answer_problem(self, correctness):
+        """
+        Problem is not set up to work.
+        """
+        # No tests are written yet for actually submitting a custom JSInput problem.=
+        pass
```

Comment:
Do you think we should use

`raise SkipTest("Test incompatible with the current problem type")` 

with a message the works for this problem, that way it will show up as skipped in the report?